appcues/src/main/java/com/appcues/data/mapper/step/StepMapper.kt

```diff
@@ -26,11 +29,16 @@ internal class StepMapper(
         val stepTraits = from.traits.map { it to STEP }
         val mergedTraits = stepTraits.mergeTraits(stepContainerTraits)
         val mergedActions = from.actions.mergeActions(stepContainerActions)
+        val mappedTraits = traitsMapper.map(mergedTraits)
 
         return Step(
             id = from.id,
             content = from.content.mapPrimitive(),
-            stepDecoratingTraits = traitsMapper.map(mergedTraits).filterIsInstance(StepDecoratingTrait::class.java),
+            // TODO only one of each type, eg: @appcues/backdrop @appcues/skippable etc
```

Comment:
as part of future task we will add some logic here maybe? 

we need to add a check somewhere, maybe in the mappers or in the moshi adapter to scan traits and look for traits of same type in the same "level"